README.md

```diff
@@ -1,12 +1,12 @@
 # Data Gemma
 
-This repo contains client library code for accessing [DataGemma](paper link), an
+This repo contains client library code for accessing DataGemma, an
 open model that helps address the challenges of hallucination by grounding LLMs
 in the vast, real-world statistical data of Google's Data Commons.
 
 There are two methodology to achieve this: Retrieval Interleaved Generation
```

Comment:
should this be updated to ".. two methodologies.."?

Reply:
thanks for catching that!